src/components/cutscenes/PokeCenterCutscene.ts

```diff
@@ -177,6 +178,13 @@ export class PokeCenterCutscene<TGameStartr extends FullScreenPokemon> extends C
                 this.gameStarter.scenePlayer.stopCutscene();
             });
         this.gameStarter.menuGrapher.setActiveMenu("GeneralText");
+        const mapLocation: string = this.gameStarter.itemsHolder.getItem("map");
+        const mapInfo: IMap = this.gameStarter.areaSpawner.getMap() as IMap;
+        const location: string | undefined = mapInfo.locationDefault;
```

Comment:
If you renamed `mapLocation` to `map`, you could inline both into the `setItem` call as `{ location, map }`.